django/contrib/admin/static/admin/js/nav_sidebar.js

```diff
@@ -3,12 +3,15 @@
     const toggleNavSidebar = document.getElementById('toggle-nav-sidebar');
     if (toggleNavSidebar !== null) {
         const navLinks = document.querySelectorAll('#nav-sidebar a');
+        const navSidebar = document.getElementById('nav-sidebar');
         function disableNavLinkTabbing() {
+            navSidebar.style.visibility = 'hidden';
             for (const navLink of navLinks) {
                 navLink.tabIndex = -1;
             }
         }
         function enableNavLinkTabbing() {
+            navSidebar.style.visibility = 'visible';
```

Comment:
Setting `tabIndex` on both sidebar links and filter shouldn't be necessary anymore :thinking: So we should get rid of `disableNavFilterTabbing()`/`enableNavFilterTabbing()`, and simplify `disableNavLinkTabbing()`/`enableNavLinkTabbing()`.

Reply:
Sorry for the slow response and thanks for the feedback. Pr updated!

Reply:
We should avoid inline styles, so I moved this to the `nav_sidebar.css`.

Reply:
Much better! thanks for the updates